src/sass/_wheel.scss

```diff
@@ -22,22 +23,57 @@ main aside {
       }
     }
   }
+
+  .possible-prizes {
+    position: absolute;
+    word-wrap: break-word;
+    width: 20%;
+    top: 0;
+    right: 0;
+    flex-direction: column;
+    
+    p {
+      font-size: 2rem;
+    }
+    span {
+      font-size: 1.5rem;
+      padding: .2rem .5rem;
+    }
+  }
 }
 
 .wheel-img {
-  width: 70%;
-  &:hover {
-    transition: 20s;
-    transform: rotate(720deg);
-  }
+  width: 80%;
+  padding: 20px 20px 20px 5px;
 }
 
-h4 {
+.prize-container {
+  position: absolute;
+  text-align: center;
+  white-space: nowrap;
+  width: 350px;
+  top: 35%;
+  left: 5%;
   font-family: $bodyfont;
   font-size: 3rem;
 }
 
 
+.prize-animation {
+  animation: spin-prize;
+  animation-duration: 3s;
+}
+
+@keyframes spin-prize {
```

Comment:
i think this is an awesome animation but i think we should define it in our variables sass file with the other animation!

Reply:
Fixed! moved keyframs into sass vars file